packaging/redhat/zeromq.spec

```diff
@@ -67,7 +67,18 @@ BuildRequires:  gnutls-devel
 %else
 %define TLS no
 %endif
-BuildRequires: gcc, make, gcc-c++, libstdc++-devel, asciidoc, xmlto
+%if 0%{?rhel_version}
+%if 0%{?rhel_version} > 700
```

Comment:
why the nested if? second one should be enough? Also you'll want `>= 800` IIRC that's better

Reply:
the problem is that we need to deal with:
* all non-RHEL cases
* RHEL < 8 case
which are 2 conditions...

In other words:

```
%if 0%{?rhel_version} >= 800
BuildRequires:  asciidoctor
%else
# this could be both the RHEL7 case or just a non-RHEL case
%endif
```

I can try with:

```
%if 0%{?rhel_version} >= 800 || ! 0%{?rhel_version}
BuildRequires:  asciidoctor
%else
# this should be only the RHEL7 case, so this branch can be empty
%endif
```

but I'm not sure if that's much more readable...

Reply:
Ah yeah, never mind then